### DISCUSSION TITLE
Live Preview

### DISCUSSION THREAD

**Discussion by coder3112** [rust-windowing/winit#2923] [Q&A]
I am using WGPU + Winit and am wondering whether a way exists to reload changes in a live preview. The reason I ask here is that I am using the event loop provided by Winit, so any hot reloads would have to start there. 

PS: I have not tried anything yet, I do not even have any idea where I would begin.

**Comment by kchibisov** [comment]
I'm not sure what you're trying to live reload? Could you explain in particular what you want to do? Shaders, entire code, something else?
